src/web/mjs/connectors/LineMangaJP.mjs

```diff
@@ -62,6 +62,10 @@ export default class LineMangaJP extends Connector {
                 });
             });
         }
+        if (option.isPortal) {
+            return Object.values(option.portalPages)
+                .map(pp => this.createConnectorURI({ pp }));
```

Comment:
May use a more expressive variable name instead of `pp`

Reply:
Also it seems `pp` is not used, only `pp.url` is used, so instead of providing `pp` as parameter, may provide `pp.url` as parameter for connectorURI